spec/models/game_spec.rb

```diff
@@ -63,4 +63,12 @@
       end
     end
   end
+
+ describe "#populate_board" do
+    it "should populate board only if both black and white player ids are present" do
+      game = Game.create(white_player_id: 0, black_player_id: 1)
```

Comment:
My guess is that this is why the tests are failing - the piece classes have a validation that ensures that their user exists.

You should probably add something like:

```ruby
white_user = User.create(... dummy details ...)
black_user = User.create(... dummy details ...)
game = Game.create(white_player_id: white_player.id, black_player_id: black_player.id)
```